src/components/CreateAppModal/CreateAppModal.jsx

```diff
@@ -1,21 +1,21 @@
+import PropTypes from 'prop-types';
 import React, { useState } from 'react';
-import { useDispatch } from 'react-redux';
 import { Formik, ErrorMessage, Field } from 'formik';
 import * as Yup from 'yup';
 import { Button, Modal, Form } from 'react-bootstrap';
 
-import { addApp } from '../../ducks/apps/all';
-
 const AppSchema = Yup.object().shape({
   name: Yup.string()
     .min(5, 'Too Short!')
     .max(50, 'Too Long!')
     .required('Name is required'),
 });
 
-const CreateAppModal = () => {
-  const dispatch = useDispatch();
+const propTypes = {
+  onSubmit: PropTypes.func.isRequired,
+};
 
+const CreateAppModal = (props) => {
```

Comment:
better to use destructuring
```suggestion
const CreateAppModal = ({onSubmit}) => {
```